tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist =
+    py27,py37,py38
+
+[testenv]
+deps =
+    .[test]
+commands =
+    zope-testrunner --test-path=src
```

Comment:
Although we do not use pytest here, I would still like to check with flake8. Maybe you could implement a linting env similar to https://github.com/zopefoundation/Zope/blob/master/tox.ini#L61 with the basic flake8 tests.